requirements/engine.txt

```diff
@@ -0,0 +1,14 @@
+albumentations>=1.1.0
```

Comment:
or `base`?

let's say if a user just want the basic installation that only performs the training inference capabilities, which option would be better

```bash
anomalib install base
anomalib install basic
anomalib install core
```

Also, it might be an idea to discuss the other options as well?

`anomalib install openvino?` or `anomalib install inference`?

Reply:
I thought `core` was an appropriate name, so I renamed this to `core.txt`.
Later on, as @samet-akcay says, we need to break core into smaller pieces. Like an `inference.txt` just for inference. Instead, we should look at the import structure of the whole library.

Reply:
https://github.com/openvinotoolkit/anomalib/pull/1748/commits/95ebf8d788175760a772387913fa44163e370de7